### ISSUE TITLE
"Unexpected token: " when having empty comment on a type.

### ISSUE THREAD

**Issue by jeggy** [jimkyndemeyer/js-graphql-intellij-plugin#408]
**Describe the bug**
When a type has a empty comment and then a field within this type has some comment. It fails to read anything that comes afterwards.

**To Reproduce**
1. Have a empty comment on a type
2. Have some comment on a field inside this type
3. Example below:
```graphql
""""""
type ServerType {
    """My comment"""
    valueFromServer: String!
}
```

**Expected behavior**
Should just work as if there was no empty comment on the type.

**Screenshots**

![image](https://user-images.githubusercontent.com/51459/97108904-59280f80-16c8-11eb-884b-cfcebb9f3a74.png)

**Version and Environment Details**
**Operation system**: Pop!OS 20.04
**IDE name and version**: IntelliJ IDEA 2020.2.3 (Ultimate Edition)
**Plugin version**: 2.6.0

**Additional context**
The [apollo tooling](https://github.com/apollographql/apollo-tooling/) `client:download-schema` command generates a file that looks like this. 

I have temporarly fixed this issue, by running a script afterwards that replaces all `""""""` with `""" """`.

**Comment by mernxl** [comment]
> I have temporarly fixed this issue, by running a script afterwards that replaces all """""" with """ """.

In my case, i fixed it by running **prettier** to format the schema generated by `apollo client:download-schema`

I have this in my package.json, scripts section.
```json
  ...
  "prettier:fix-schema": "prettier --parser graphql --write 'schema.graphql'",
  "apollo:download": "apollo client:download-schema -c apollo.config.js schema.graphql && yarn run prettier:fix-schema",
  ...
```

**Comment by vepanimas** [comment]
Fixed in 3.0.0.
